src/test_utils.rs

```diff
@@ -44,7 +44,10 @@ pub fn setup_test_config() -> Result<BuilderConfig> {
         oauth_token_refresh_interval: 300, // 5 minutes
         builder_helper_address: Address::default(),
         concurrency_limit: 1000,
-        start_timestamp: 1740681556, // pecorino start timestamp as sane default
+        slot_calculator: SlotCalculator::new(
+            1740681556, // pecorino start timestamp as sane default
+            0, 1,
```

Comment:
should these be constants?

Reply:
I don't know what these are, so i just moved them as is. @dylanlott can make a change here